core/domain/platform_parameter_list.py

```diff
@@ -85,8 +86,12 @@ class ParamNames(enum.Enum):
     ' page.',
     platform_parameter_domain.FeatureStages.PROD)
 
-
 Registry.create_feature_flag(
     ParamNames.DIAGNOSTIC_TEST,
     'This flag is for the diagnostic test functionality.',
     platform_parameter_domain.FeatureStages.PROD)
+
+Registry.create_feature_flag(
+    ParamNames.SIGNIN_WITH_POPUP,
+    'This flag is for the new signin with popup feature.',
```

Comment:
Actually, thinking about this a bit more, this description could have a bit more specificity to it. As of now, the feature is 'new' however that won't always be true. Instead, the description should describe the specific functionality being configured by this flag.

Reply:
Yeah, that sounds reasonable